content-src/components/Header/Header.js

```diff
@@ -8,7 +8,8 @@ const Header = React.createClass({
   render() {
     const props = this.props;
     const currentRoute = props.currentRoute || {};
-    return (<header className="head">
+    return (<header className="head" hidden={props.currentRoute.path === "/"}>
```

Comment:
Curious, why wouldn't we want this shared header in both pages (since it displays my username and FxA avatar)?